spec/integration/solo/solo_spec.rb

```diff
@@ -28,7 +28,7 @@
       file "config/solo.rb", <<~EOM
         chef_repo_path "#{@repository_dir}"
       EOM
-      result = shell_out("bundle exec chef-solo -c \"#{path_to("config/solo.rb")}\" -l debug", cwd: chef_dir)
+      result = shell_out("#{chef_solo} -c \"#{path_to("config/solo.rb")}\" -l debug", cwd: chef_dir)
```

Comment:
```suggestion
      result = shell_out("#{chef-solo} -c \"#{path_to("config/solo.rb")}\" -l debug", cwd: chef_dir)
```

Reply:
This is correct as-is - it's referring to the `let(:chef_solo)` above